common/app/services/OphanApi.scala

```diff
@@ -1,30 +1,46 @@
 package services
 
+import common.editions.{Au, Uk, Us, International}
+
 import java.net.URLEncoder
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import java.time.temporal.ChronoUnit
-
-import common.{BadConfigurationException, GuLogging}
+import common.{BadConfigurationException, Edition, GuLogging}
 import conf.Configuration._
 import play.api.libs.json._
 import play.api.libs.ws.WSClient
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration.DurationInt
 
+// Base on this response: https://dashboard.ophan.co.uk/api/mostread
 case class OphanMostReadItem(url: String, count: Int)
 object OphanMostReadItem {
   implicit val jsonReads = Json.reads[OphanMostReadItem]
 }
 
+// Based on this response: https://dashboard.ophan.co.uk/api/deeplyread
+case class OphanDeeplyReadItem(path: String, benchmarkedAttentionTime: Int)
+object OphanDeeplyReadItem {
+  implicit val jsonReads = Json.reads[OphanDeeplyReadItem]
+}
+
 class OphanApi(wsClient: WSClient)(implicit executionContext: ExecutionContext)
     extends GuLogging
     with implicits.WSRequests {
   private val mostViewedDateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd")
 
   private def ensureHostSecure(host: String): String = host.replace("http:", "https:")
 
+  private def countryFromEdition(edition: Edition) =
+    edition match {
```

Comment:
Editions have a `networkFrontId` field that has the same values as below, worth using that?

Reply:
These are specific to the Ophan API - so wouldn't want to couple them in case they drift and this will potentially break.